frontend/src/components/NewPassword/NewPassword.js

```diff
@@ -22,14 +27,43 @@ class NewPassword extends Component {
     this.setState({ [event.target.name]: event.target.value });
   };
 
+  handleSubmit = (event, changeCreatedState) => {
+    event.preventDefault();
+    const { history } = this.props;
+    const { password, confirmPassword } = this.state;
+    const { resetPasswordToken } = this.state;
+    const queryString = `resetpassword/?q=${resetPasswordToken}`;
+    if (password !== confirmPassword) {
+      alert("Passwords don't match"); // eslint-disable-line no-undef
+    } else {
+      resetPassword(this.state)
+        .then((response) => {
+          changeCreatedState(response.status);
+          history.push(queryString);
+        })
+        .catch((error) => {
+          if (error.message.includes('422')) {
+            const parentForm = document.getElementsByTagName('form')[0];
+            const errorDiv = document.createElement('p');
+            errorDiv.setAttribute('class', 'errorMessage reset-password');
+            errorDiv.innerHTML = 'Please enter a valid password.';
+            parentForm.insertAdjacentElement('beforeend', errorDiv);
```

Comment:
If we're going to copy this from that other page, we should probably use the updated version from SignIn to avoid repeated errors.

Reply:
See https://github.com/the-difference-engine/rescue-leftover-cuisine2/blob/staging/frontend/src/components/SignIn/SignIn.js#L37-L40 and https://github.com/the-difference-engine/rescue-leftover-cuisine2/blob/staging/frontend/src/components/SignIn/SignIn.js#L97